package.json

```diff
@@ -52,7 +54,9 @@
     "mocha": "^8.3.1",
     "npm-run-all": "^4.1.5",
     "rollup": "^2.41.2",
-    "shelljs": "^0.3.0"
+    "shelljs": "^0.3.0",
+    "typescript": "^4.5.4",
+    "unicode-6.3.0": "^0.7.5"
```

Comment:
I think we're not using this dependency.

Reply:
We're using it in `tests/lib/libraries.js`